packages/node_modules/@ciscospark/spark-core/src/lib/credentials/credentials.js

```diff
@@ -420,6 +420,23 @@ const Credentials = SparkPlugin.extend({
       })
       .then(() => {
         this.scheduleRefresh(this.supertoken.expires);
+      })
+      .catch((error) => {
+        const {InvalidRequestError} = grantErrors;
```

Comment:
should this be defined above? don't need to set it in a variable every time it hits the catch